plugins/pre-commit/pre-commit.plugin.zsh

```diff
@@ -0,0 +1,11 @@
+# Aliases for pre-commit
+alias pc='pre-commit'
```

Comment:
Conflicts with https://github.com/ohmyzsh/ohmyzsh/blob/master/plugins/macports/README.md

Reply:
Thanks for the hint. I will think about an alternative base alias for pre-commit.

In general, are all plugin combinations collision free? And is there a CI job that validates that?

Reply:
AFAIK, yes. And no, there aren't any automatic systems for avoiding collisions.

Reply:
@cbachhuber Any suggestion for a good alias here?

Reply:
Good question, how about

- `pct`
- `pcmt`
- `prcmt`

I don't see aliases in the repo for any of these options, and no similarly named Ubuntu package stands out to me. Do you like any of these?

Reply:
I also thought about `pct` because of brevity.

But should we use this base for all other aliases as well or stick to them as they are. Changing them would be a bit more consistent but also more to type. 

I would go for only changing `pc` to `pct`

Reply:
I'm torn - I would like to avoid this inconsistency in the aliases. Thus, I'd prefer using `pct` as base for all aliases. @LunarWatcher, what do you think?

Reply:
`pct` sounds good to me. I agree with using it as a prefix for consistency; this should also let us avoid conflicts in the future. 

Alternatively, there's `prc` (**pr**e-**c**ommit), though I haven't properly checked it for package conflicts. The second advantage with `prc`, beyond using the early characters, is that there is 0 overlap with existing commands, beyond the `p`-based ones. There's nothing that starts on `pr` at the moment, though.

But I'm fine with either of these; pick whichever you feel is more natural

Reply:
I updated the aliases consistently to use the base `prc`.